src/assets/data/localisations.ts

```diff
@@ -623,6 +623,21 @@ export const Localisations = {
     pl: 'Proszę czekać',
     hb: 'המתן בבקשה'
   },
+  BTN_SETTING_LOGOUT_CANCEL: {
+    en: "Cancel"
+  },
+  BTN_SETTING_LOGOUT_CONFIRM:{
+    en: "Confirm"
+  },
+  SETTINGS_LOGOUT_ALERT:{
+    en: "Log out"
+  },
+  SETTINGS_LOGOUT_ALERT_DESC:{
```

Comment:
Maybe better say "Are you sure you want to logout the app? You may lose your today's progress of completed surveys!" something like that?

Reply:
Ok